extensions/ql-vscode/src/vscode-tests/no-workspace/remote-queries/variant-analysis-manager.test.ts

```diff
@@ -44,9 +44,21 @@ describe('Variant Analysis Manager', async function() {
     variantAnalysis = createMockApiResponse('in_progress', scannedRepos);
 
     try {
-      const extension = await extensions.getExtension<CodeQLExtensionInterface | Record<string, never>>('GitHub.vscode-codeql')!.activate();
-      cli = extension.cliServer;
-      variantAnalysisManager = new VariantAnalysisManager(extension.ctx, cli, storagePath, logger);
+      const ctx = createMockExtensionContext();
+      cli = createMockCliServer({
```

Comment:
Sorry if this is a silly question, but if we're having to provide a mock CLI server, does that indicate that these tests should be staying in the CLI-integration test suite?

Reply:
I don't think so, but @elenatanasoiu please correct me if I'm wrong. The `cli-integration` tests will also actually call the cli, but in these specific tests we're not actually calling the cli. The only place where the cli would be called from these tests is if we're decoding BQRS results, but that isn't something we're testing from these tests.

Reply:
Yeah, as @koesie said, we don't actually need the CLI for our tests. However, both the `variantAnalysisManager` and `variantAnalysisResultsManager` tests require some sort of CLI instance when they're first initialised so we need a dummy version of it without actually using it.